feature-group-definitions/font-palette-animation.yml

```diff
@@ -0,0 +1,5 @@
+name: font-palette animation
+spec: https://drafts.csswg.org/css-fonts-4/#font-palette-prop
+usage_stats: https://chromestatus.com/metrics/css/timeline/animated/709
+compat_features:
+  - css.types.color.palette-mix
```

Comment:
This doesn't exist and probably should. Maybe file an issue on BCD?

(I'm not stressed about missing keys getting in here and there, if we're committed to tagging in BCD.)

Reply:
Oh, I added this with the intention to add it to BCD but then forgot. I'll do that before merging this PR.

Reply:
Ah, this was a copy-and-paste failure on my part (I opened "both). Sorry!

Reply:
Added in https://github.com/mdn/browser-compat-data/pull/22242 at a different path than I originally though. I'll just merge this and we'll sort out any mismatch when we migrate `compat_features` fully.